docker/orcpub/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM demonsthere/orcpub:bootstrap-lein as builder
+MAINTAINER daemonsthere@gmail.com
+
+ENV REPO="Demonsthere"
+ENV BRANCH="master"
+
+# Get orcpub source
+WORKDIR /opt
+RUN git clone --branch ${BRANCH} https://github.com/${REPO}/orcpub.git &&\
+	cd orcpub &&\
+	lein uberjar
+
+FROM demonsthere/orcpub:datomic as runner
```

Comment:
We do not need to run Datomic in the same container. Do the following:
Merge into `docker-compose.yml`:
```
service:
  orcpub:
    depends_on:
      - datomic
    environment:
      DATOMIC_URL: "datomic:free://datomic:4334/orcpub"
  datomic:
    image: akiel/datomic-free
```
The orcpub container will still fail, due to the schema not being bootstrapped.
There is a way to bootstrap it with a `lein run -m user --init-database` type command with some modifications to the code.

Reply:
In this case, please derive the runner from `openjdk:8-jre-alpine` or `openjdk:8-jre-slim` (which is currently `stretch`)